### PR TITLE
grains, isbn-verifier: Typo fixes

### PR DESCRIPTION
**Pull request by ferhatelmas**

### CODE REVIEW

File: exercises/isbn-verifier/description.md [hunk 1]
```diff
@@ -32,4 +32,4 @@ It's getting even trickier since the check-digit of an ISBN-10 can be 'X'.
 
 * Generate a valid ISBN-13 from the input ISBN-10 (and maybe verify it again with a derived verifier)
 
-* Generate valid ISBN, maybe even from a given starting ISBN
\ No newline at end of file
+* Generate valid ISBN, maybe even from a given starting ISBN
```
Comment (petertseng) [inline exercises/isbn-verifier/description.md]: I wouldn't say adding newline is considered fixing typo

File: exercises/isbn-verifier/description.md [hunk 2]
```diff
@@ -2,7 +2,7 @@ Check if a given ISBN-10 is valid.
 
 ## Functionality
 
-Given an unkown string the program should check if the provided string is a valid ISBN-10.
```
Comment (petertseng) [inline exercises/isbn-verifier/description.md]: http://cf.geekdo-images.com/images/pic1453075.jpg

File: exercises/grains/canonical-data.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "comments": [
     "The final tests of square test error conditions",
-    "The expection for these tests is -1, indicating an error",
+    "The exception for these tests is -1, indicating an error",
```
Comment (petertseng) [inline exercises/grains/canonical-data.json]: I think it is expectation rather than exception

**Comment by ferhatelmas** [comment]
@petertseng reverted eol and changed other as your suggestion since I had already hesitated about it.

File: exercises/isbn-verifier/description.md (hunk 1 above)
Comment (ferhatelmas) [reply exercises/isbn-verifier/description.md]: Right reverted.

File: exercises/grains/canonical-data.json (hunk 3 above)
Comment (ferhatelmas) [reply exercises/grains/canonical-data.json]: Possible :smile:

File: exercises/isbn-verifier/description.md (hunk 2 above)
Comment (ferhatelmas) [reply exercises/isbn-verifier/description.md]: In every exchange with you, I learn something.

**Pull request merged**